mreg_cli/main.py

```diff
@@ -241,5 +214,46 @@ def get_prompt_message():
                 print(e)
 
 
+# Define a function that returns the prompt message
+def get_prompt_message(args: argparse.Namespace, config: MregCliConfig) -> HTML:
+    """Return the prompt message."""
+    manager = OutputManager()
+    args_map = vars(args)
+
+    def fmt_prompt(prompt: str) -> str:
+        host = str(config.get("url")).replace("https://", "").replace("http://", "")
+        user = args_map.get("user") or config.get("user", "?")
+        return prompt.format(
+            # Available variables for the prompt:
+            user=user,
+            host=host,
+        )
+
+    # Try to get prompt from args -> config -> default
+    if (args_prompt := args_map.get("prompt")) is not None:
+        prompt = str(args_prompt)
+    elif (config_prompt := config.get_prompt()) is not None:
+        prompt = config_prompt
+    else:
+        prompt = DEFAULT_PROMPT
```

Comment:
This doesn't look _great_, but we have to guard against both `args.prompt` and `config["prompt"]` being None.

Reply:
Could call it "prefix" or something.